evm_arithmetization/src/cpu/kernel/constants/mod.rs

```diff
@@ -394,20 +396,26 @@ pub mod cancun_constants {
         hex!("000000000000000000000000000000001666c54b0a32529503432fcae0181b4bef79de09fc63671fda5ed1ba9bfa07899495346f3d7ac9cd23048ef30d0a154f"), // y_im
     ];
 
+    pub const HISTORY_BUFFER_LENGTH: (&str, u64) = ("HISTORY_BUFFER_LENGTH", 8191);
+
+    // Beacon constants
+    ///////////////////
```

Comment:
I remember we all spoke about this before but I genuinely can't remember what we agreed. I think I should write a style guide.

Reply:
Yeah we don't use this style in `evm_arithmetization` at all. I'd ideally like some uniformity but well, non-blocking.